bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/VectorStyle.jsx

```diff
@@ -110,7 +112,25 @@ export const VectorStyle = LocaleConsumer(({ layer, getMessage, controller, exte
     return (
         <FullWidthSpace direction='vertical'>
             <Space direction='horizontal'>
-                <Button onClick={ () => addStyle('editor', { featureStyle: {} }) }>
+                <Button
+                    onClick={ () => {
+                        const style = {
+                            ...OSKARI_BLANK_STYLE,
+                            fill: {
+                                ...OSKARI_BLANK_STYLE.fill,
+                                color: theme.color.primary
+                            },
+                            image: {
+                                ...OSKARI_BLANK_STYLE.image,
+                                fill: {
+                                    ...OSKARI_BLANK_STYLE.image.fill,
+                                    color: theme.color.primary
+                                }
+                            }
+                        };
```

Comment:
This is getting duplicated on multiple places. We should rather add:
```
import { generateBlankStyle } from 'oskari-ui/components/StyleEditor/index';
```
and do 
```suggestion
                        const style = generateBlankStyle(theme);
```

Reply:
```
export const generateBlankStyle = (theme) => {
    return {
        ...OSKARI_BLANK_STYLE,
        fill: {
            ...OSKARI_BLANK_STYLE.fill,
            color: theme.color.primary
        },
        image: {
            ...OSKARI_BLANK_STYLE.image,
            fill: {
                ...OSKARI_BLANK_STYLE.image.fill,
                color: theme.color.primary
            }
        }
    };
};